src/compiler/rtlib.jl

```diff
@@ -44,7 +44,7 @@ function load_device_libs(agent)
         "hip.amdgcn.bc",
         "irif.amdgcn.bc",
         "ockl.amdgcn.bc",
-        "oclc_isa_version_803.amdgcn.bc", # FIXME: Load based on agent name!
+        "oclc_isa_version_906.amdgcn.bc", # FIXME: Load based on agent name!
```

Comment:
Note I modified 803 to 906...change back to your dev when debugging

Reply:
Yep that's totally fine for now; although if you're feeling adventurous, we should fix this to load whatever agent ISA is in the job struct.